examples/playground/panel-lite-editor.js

```diff
@@ -0,0 +1,131 @@
+const appContainer = "app-container"
+const codeEditor = "code-editor"
+
+const welcomeExample=`\
+import panel as pn
+pn.extension()
+
+pn.pane.Markdown("Welcome! Try clicking the \`run\` button.").servable()
+`
+
+const buttonExample=`\
+import panel as pn
+pn.extension(design="material")
+button = pn.widgets.Button(name="Click Me")
+component = pn.Row(button, pn.bind(lambda c: c, button.param.clicks))
+
+component.servable()
+`
+
+const matplotlibExample=`
+import panel as pn
+
+import numpy as np
+
+from matplotlib.figure import Figure
+from matplotlib import cm
+from matplotlib.backends.backend_agg import FigureCanvas  # not needed for mpl >= 3.1
+
+Y, X = np.mgrid[-3:3:100j, -3:3:100j]
+U = -1 - X**2 + Y
+V = 1 + X - Y**2
+speed = np.sqrt(U*U + V*V)
+
+fig0 = Figure(figsize=(8, 6))
+ax0 = fig0.subplots()
+FigureCanvas(fig0)  # not needed for mpl >= 3.1
+
+strm = ax0.streamplot(X, Y, U, V, color=U, linewidth=2, cmap=cm.autumn)
+fig0.colorbar(strm.lines)
+
+pn.pane.Matplotlib(fig0, dpi=144).servable()
+`
+
+const perspectiveExample=`
```

Comment:
## Todo

- [ ] Fix  sporadic `pyodide.ffi.JsException: Error: Error rendering Bokeh model: could not find ... HTML tag`
  - I've experienced this when click my way through the examples from left to right.

<details>
<summary>pyodide.ffi.JsException: ... could not find ... HTML tag`</summary>

```bash
Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python311.zip/_pyodide/_base.py", line 571, in eval_code_async
    await CodeRunner(
  File "/lib/python311.zip/_pyodide/_base.py", line 396, in run_async
    await coroutine
  File "<exec>", line 45, in <module>
  File "/lib/python3.11/site-packages/panel/io/pyodide.py", line 519, in write_doc
    views = await Bokeh.embed.embed_items(JSON.parse(docs_json), JSON.parse(render_items))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pyodide.ffi.JsException: Error: Error rendering Bokeh model: could not find #e97f9bbb-8b5e-42a7-b685-b355a0ee7b92 HTML tag

    at new_error (pyodide.asm.js:9:12519)
    at pyodide.asm.wasm:0x158827
    at pyodide.asm.wasm:0x15fcd5
    at _PyCFunctionWithKeywords_TrampolineCall (pyodide.asm.js:9:123052)
    at pyodide.asm.wasm:0x1a3091
    at pyodide.asm.wasm:0x289e4d
    at pyodide.asm.wasm:0x1e3f77
    at pyodide.asm.wasm:0x1a3579
    at pyodide.asm.wasm:0x1a383a
    at pyodide.asm.wasm:0x1a38dc
    at pyodide.asm.wasm:0x2685c5
    at pyodide.asm.wasm:0x26e3d0
    at pyodide.asm.wasm:0x1a3a04
    at pyodide.asm.wasm:0x1a3694
    at pyodide.asm.wasm:0x15f45e
    at Module.callPyObjectKwargs (pyodide.asm.js:9:81732)
    at Module.callPyObject (pyodide.asm.js:9:82066)
    at wrapper (pyodide.asm.js:9:58562)
```
</details>